exchanges/bitstamp/bitstamp_test.go

```diff
@@ -411,14 +427,16 @@ func TestGetWithdrawalRequests(t *testing.T) {
 func TestGetUnconfirmedBitcoinDeposits(t *testing.T) {
 	t.Parallel()
 
-	_, err := b.GetUnconfirmedBitcoinDeposits(context.Background())
-	switch {
-	case sharedtestvalues.AreAPICredentialsSet(b) && err != nil && !mockTests:
-		t.Error("GetUnconfirmedBitcoinDeposits() error", err)
-	case !sharedtestvalues.AreAPICredentialsSet(b) && err == nil && !mockTests:
-		t.Error("Expecting an error when no keys are set")
-	case mockTests && err != nil:
-		t.Error("GetUnconfirmedBitcoinDeposits() error", err)
+	if !mockTests {
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
+	}
+	d, err := b.GetUnconfirmedBitcoinDeposits(context.Background())
+	assert.NoError(t, err, "GetUnconfirmedBitcoinDeposits should not error")
+	if mockTests {
+		assert.NotEmpty(t, d, "Deposits should not be empty")
+		for _, res := range d {
+			assert.Equal(t, "0x6a56f5b80f04b4fd70d64d72e1396698635e5436", res.Address)
```

Comment:
This + the docs example made me laugh, "GetUnconfirmedBitcoinDeposits" but returns an ETH address!